mods/d2k/rules/infantry.yaml

```diff
@@ -113,6 +113,40 @@ medic:
 	AttractsWorms:
 		Intensity: 180
 
+grenadier:
```

Comment:
Could you please move this back to its original location so that the diff shows what has actually changed?